src/System.Data.Odbc/src/Common/System/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Data.Common
+{
+    internal static class DbConnectionStringDefaults
```

Comment:
Yeah, you will have a bunch of refactoring to take care of. I thought I will point the issue out now itself. 
It would be good to create separate issues for these comments.
About the Common folder, you can drop the files in https://github.com/dotnet/corefx/tree/master/src/Common/src/System/Data/Common 
I had placed some classes for System.Data here. 
Keep the folder structure in Common, the same as how it is, in the current assemblies. 

Example of references to files in Common folder 
https://github.com/dotnet/corefx/blob/master/src/System.Data.SqlClient/src/System.Data.SqlClient.csproj#L49